nemesis-generator/configuration_generator/__main__.py

```diff
@@ -1,23 +1,28 @@
 import argparse
 import secrets
+import time
 
 import yaml
 from symbolchain.CryptoTypes import Hash256, PrivateKey
 from symbolchain.facade.NemFacade import NemFacade
+from symbolchain.nem.Network import Network
 from zenlog import log
 
 MICROXEM_PER_XEM = 1000000
 
 
 class NemesisConfigurationGenerator:
-	def __init__(self, network):
-		self.facade = NemFacade(network)
+	def __init__(self, network_name, network_identifier):
+		self.epoch_time = int(time.time_ns() / 1000000000)
```

Comment:
will this break this from generating a well known network (i.e. TESTNET) ?

Reply:
yes, I thought for each testnet we wanted a new epoch time.  Is this not needed for NEM?
if so then I use the default epoch time

Reply:
i think the "main" testnet epoch will probably come from SDK, so would be good for that to work at least